bmcweb.service.in

```diff
@@ -13,6 +13,8 @@ ExecReload=kill -s HUP $MAINPID
 ExecStart=@MESON_INSTALL_PREFIX@/bin/bmcweb
 Type=simple
 WorkingDirectory=/home/root
+StartLimitInterval=15
```

Comment:
Also, I did some searching around, we changed the BMC behavior to 2 in 30s (https://grok.openbmc.org/xref/openbmc/openbmc/meta-phosphor/recipes-core/systemd/phosphor-systemd-policy/service-restart-policy.conf?r=0e72d71a&mo=745&fi=16#16).  Done via https://gerrit.openbmc-project.xyz/c/openbmc/openbmc/+/8400 because core files on AST2500 machines were slowing things down so much. Given your test results, my vote would be we do 5 in 20s. That's still reducing the time by 10s and giving us 3 more resets but gives us a little more room on the core file crash scenario. Your test was on the hairy edge of the 15s.

Reply:
All seem reasonable. Done. 
Testing again.

Reply:
When I tested the 2nd time: 
```
# journalctl | grep systemd-coredump | grep Process
May 16 20:43:06 perfrain86bmctest systemd-coredump[1961]: Process 1668 (bmcweb) of user 0 dumped core.
May 16 20:43:08 perfrain86bmctest systemd-coredump[1999]: Process 1994 (bmcweb) of user 0 dumped core.
May 16 20:43:10 perfrain86bmctest systemd-coredump[2036]: Process 2031 (bmcweb) of user 0 dumped core.
May 16 20:43:16 perfrain86bmctest systemd-coredump[2137]: Process 2116 (bmcweb) of user 0 dumped core.
May 16 20:43:25 perfrain86bmctest systemd-coredump[2472]: Process 2453 (bmcweb) of user 0 dumped core.
May 16 20:43:27 perfrain86bmctest systemd-coredump[2700]: Process 2695 (bmcweb) of user 0 dumped core.
May 16 20:43:29 perfrain86bmctest systemd-coredump[2716]: Process 2711 (bmcweb) of user 0 dumped core.
May 16 20:43:31 perfrain86bmctest systemd-coredump[2732]: Process 2721 (bmcweb) of user 0 dumped core.
May 16 20:43:32 perfrain86bmctest systemd-coredump[2742]: Process 2737 (bmcweb) of user 0 dumped core.
May 16 20:43:34 perfrain86bmctest systemd-coredump[2752]: Process 2747 (bmcweb) of user 0 dumped core.
May 16 20:43:46 perfrain86bmctest systemd-coredump[2898]: Process 2777 (bmcweb) of user 0 dumped core.

May 16 20:43:47 perfrain86bmctest systemd[1]: bmcweb.socket: Failed with result 'service-start-limit-hit'.
```

Pretty slow between restarts there.. Due to this I propose 4/30